plugin/rewrite/README.md

```diff
@@ -209,12 +209,21 @@ rewrites the first local option with code 0xffee, setting the data to "abcd". Eq
 * A variable data is specified with a pair of curly brackets `{}`. Following are the supported variables:
   {qname}, {qtype}, {client_ip}, {client_port}, {protocol}, {server_ip}, {server_port}.
 
-Example:
+* If metadata plugin is enabled, then labels are supported as variables if they are presented within curly brackets. 
+the variable data will be filled with the value associated with that label. If that label is not provided, 
+the variable will be silently substitute by an empty string.  
+
+Examples:
 
 ~~~
 rewrite edns0 local set 0xffee {client_ip}
 ~~~
 
+~~~
```

Comment:
The 2nd example needs more docs, something, ala:

The following example uses the imaginary *mac* plugin to retrieve the mac address, it adds a label `mac/address`, this can be etc. etc.

Reply:
ok